akka-persistence/src/test/scala/akka/persistence/EndToEndEventAdapterSpec.scala

```diff
@@ -228,7 +228,7 @@ abstract class EndToEndEventAdapterSpec(journalName: String, journalConfig: Conf
         .withoutPath(s"""$journalPath.event-adapter-bindings."${classOf[EndToEndEventAdapterSpec].getCanonicalName}$$A"""")
 
       withActorSystem("MissingAdapterSystem", journalConfig.withFallback(missingAdapterConfig)) { implicit system2 ⇒
-        EventFilter[ActorInitializationException](occurrences = 1, pattern = ".*undefined event-adapter.*") intercept {
+        EventFilter[ActorInitializationException](occurrences = 1, pattern = ".*akka.persistence.journal.leveldb.*") intercept {
```

Comment:
hm, should we unwrap InvocationTargetException since that is not very useful for anyone?